packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -34,6 +34,223 @@ export interface HttpRouteMatch {
    * and you want the route to match requests to my-service.local/metrics, your prefix should be /metrics.
    */
   readonly prefixPath: string;
+
+  /**
+   * Specifies the client request headers to match on. All specified headers
+   * must match for the route to match.
+   *
+   * @default - do not match on headers
+   */
+  readonly headers?: HeaderMatch[];
+
+  /**
+   * The client request method to match on. Specify only one.
```

Comment:
What does "specify only one" means in this case?

I'd also mention explicitly this is the "HTTP request method", for clarity.

Reply:
Thanks for catching this. "Specify only one" can be removed because we're using an enum instead of a string, so there's no need to be explicit in the docs that only one HTTP method can be matched on - it's a holdover from the [CFN docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-route-httproutematch.html#cfn-appmesh-route-httproutematch-method).